fastlane/Fastfile

```diff
@@ -70,9 +70,26 @@ platform :android do
     )
 
     cleanup_fastlane_release_notes()
+    annotate_release()
 
   end
 
+  desc "Annotate release"
+  lane :annotate_release do
```

Comment:
Nit: Since we shouldn't have to call this lane directly, we could use `private_lane` instead of `lane`, and later promote it to a public-facing lane if it was needed.

Reply:
ah shoot, missed this comment. Will do in a follow up. sorry about that.